src/Shouldly/Configuration/FindMethodUsingAttribute.cs

```diff
@@ -11,6 +11,10 @@ public TestMethodInfo GetTestMethodInfo(StackTrace stackTrace, int startAt = 0)
         {
             foreach (var frame in stackTrace.GetFrames().Skip(startAt))
             {
+                if (frame == null)
+                {
+                    continue;
+                }
```

Comment:
Because I moved to .NET 5 nullability annotations, `GetFrames()` is now properly annotated so that `frame` is known at compile time to never be null.

Reply:
Same thing in another file.

Reply:
This is what you look for:
![image](https://user-images.githubusercontent.com/8040367/103154970-97de5280-4769-11eb-9147-1a70aef67e40.png)

Reply:
Also notice
![image](https://user-images.githubusercontent.com/8040367/103155034-06231500-476a-11eb-891b-824844b36bf2.png)

Reply:
Resolved in https://github.com/shouldly/shouldly/pull/740